test/python/utils.py

```diff
@@ -20,11 +28,271 @@ def tarantool_version():
 def tarantool_enterprise_is_used():
     return tarantool_version().startswith('Tarantool Enterprise')
 
+
 def create_project(module_tmpdir, project_name, template):
-    cmd = [os.path.join(basepath, "cartridge"), "create",
+    cmd = [
+        os.path.join(basepath, "cartridge"), "create",
         "--name", project_name,
-        "--template", template]
+        "--template", template
+    ]
     process = subprocess.run(cmd, cwd=module_tmpdir)
     assert process.returncode == 0, \
         "Error during creating the project"
     return os.path.join(module_tmpdir, project_name)
+
+
+def find_archive(path, arch_ext):
+    with os.scandir(path) as it:
+        for entry in it:
+            if entry.name.endswith('.' + arch_ext) and entry.is_file():
+                return os.path.join(path, entry.name)
+
+
+def recursive_listdir(root_dir):
```

Comment:
Try to use `glob.glob('**', recursive=True)`

Reply:
It sounds good, but in fact `glob.glob('path/to/dir/**')` will return `[...'path/to/dir/subdir', .'path/to/dir/subdir/file', ...]` paths. So, this call requres some additional actions to extract relpath and it can't replace using `recursive_listdir` function.